### ISSUE TITLE
NOTE differences between Torch and TensorFlow

### ISSUE THREAD

**Issue by zaccharieramzi** [benchopt/benchmark_resnet_classif#30]
ResNet implem:
- stride > 2 (can be corrected) is done a bit different
- init of weights is different
- Empirically: SGD works much better for PyTorch (reaching 95.5%).
- Batch norm seems to be implemented differently for the 2 frameworks: it causes a difference in loss at training time.

**Comment by zaccharieramzi** [comment]
Batch norm is indeed implemented differently. See the unbiased/biased thing in the paper.

Coupled weight decay not being applied correctly in TF was the reason behind it not performing as well.
